--- /var/www/roul/wbb31/wcf/templates/captcha.tpl	2009-04-30 15:15:27.000000000 +0200
+++ ./templates_patched/captcha.tpl	2011-06-01 20:26:41.485863000 +0200
@@ -7,6 +7,9 @@
 			<div class="formFieldLabel">
 				<label for="captchaString">{lang}wcf.captcha.captchaString.title{/lang}</label>
 			</div>
+			{if MODULE_SYSTEM_RECAPTCHA && RECAPTCHA_PRIVATEKEY != '' && RECAPTCHA_PUBLICKEY != ''}
+				{include file="reCaptcha"}
+			{else}
 			<div class="formField">
 				<input type="text" class="inputText" name="captchaString" value="" id="captchaString" />
 				{if $errorField == 'captchaString'}
@@ -32,6 +35,7 @@
 				//]]>
 			</script>
 			<script type="text/javascript" src="{@RELATIVE_WCF_DIR}js/Captcha.class.js"></script>
+			{/if}
 		</div>
 	{if $enableFieldset}</fieldset>{/if}
